torch/nn/_intrinsic/qat/modules/conv_bn_relu.py

```diff
@@ -0,0 +1,227 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import math
+import torch
+from torch.nn.modules.conv import _ConvNdBase
+from torch.nn.modules.utils import _pair
+from torch.nn import init
+from torch.nn._intrinsic import ConvBn2d as NNConvBn2d
+from torch.nn._intrinsic import ConvBnReLU2d as NNConvBnReLU2d
+from torch.nn import Parameter
+import torch.nn.functional as F
+
+
+class ConvBn2d(_ConvNdBase):
+    r"""
+    A ConvBn2d module is a module fused from Conv2d and BatchNorm2d,
+    attached with FakeQuantize modules for both output activation and weight,
+    used in quantization aware training.
+
+    We combined the interface of :class:`torch.nn.Conv2d` and
+    :class:`torch.nn.BatchNorm2d`.
+
+    Implementation details: https://arxiv.org/pdf/1806.08342.pdf
+
+    Similar to :class:`torch.nn.Conv2d`, with FakeQuantize modules initialized
+    to default.
+
+    Attributes:
+        freeze_bn:
+        observer: fake quant module for output activation, it's called observer
+            to align with post training flow
+        weight_fake_quant: fake quant module for weight
+
+    """
+    __FLOAT_MODULE__ = NNConvBn2d
```

Comment:
I don't think we should use magic naming here. `_FLOAT_MODULE` is sufficient

Reply:
I'll change to `__FLOAT_MODULE` since it's a class attribute.
https://pep8.org/#descriptive-naming-styles